src/app/qgisapp.cpp

```diff
@@ -5111,8 +5111,15 @@ void QgisApp::about()
 
     if ( QString( Qgis::devVersion() ) == QLatin1String( "exported" ) )
     {
-      versionString += QStringLiteral( "%1</td><td><a href=\"https://github.com/qgis/QGIS/tree/release-%1_%2\">Release %1.%2</a></td>" )
-                       .arg( tr( "QGIS code branch" ) ).arg( Qgis::versionInt() / 10000 ).arg( Qgis::versionInt() / 100 % 100 );
+      if ( Qgis::version().endsWith( QLatin1String( "Master" ) ) )
+      {
+        versionString += QStringLiteral( "%1</td><td><a href=\"https://github.com/qgis/QGIS/tree/master\">master</a></td>" ).arg( tr( "QGIS code branch" ) );
```

Comment:
the code would be even clearer / simpler if versionString += tr( "QGIS code branch" ) was done before the if()

Reply:
You are right, thank you for pointing that out. Now it should be better, I think.